src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/runMatlabTests.m

```diff
@@ -133,13 +132,18 @@
 results = runner.run(suite);
 failed = any([results.Failed]);
 
+function preparePath(path)
+dir = fileparts(path);
+dirExists = isempty(dir) || exist(dir,'dir') == 7;
+if ~dirExists		
+    mkdir(dir);		    
+end
 
-function tapToFile = getTapResultFile(resultsDir)
+function tapFile = getTapResultFile(resultsDir)
```

Comment:
Funny, I was referring to the other variable, "resultsDir" to change. This is not a dir it is the fullpath to the desired tap file. Thats why it is confusing, you can't fopen a dir for example, so the code is misleading.

This can wait to change in a follow-up commit.